app/controllers/events/view/edit/attendee.js

```diff
@@ -38,4 +39,4 @@ export default Controller.extend(EventWizardMixin, {
       }
     }
   }
-});
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4742204)